thamos/cli.py

```diff
@@ -1364,13 +1364,27 @@ def images(
         for item in result:
             table_content = []
             for key, value in item.items():
+                if key == "package_extract_document_id":
+                    container_image_search_link = (
+                        f"{configuration.thoth_search_ui_url}image/{value}"
```

Comment:
It might worth checking that `_thoth_search_ui_url` is not None (user-api provides its value).

Reply:
Do you mean `thoth_search_ui_url`?

Reply:
Yes - if we run thamos against user-api that does not expose the given header yes, `thoth_search_ui_url` will be set to `None`.